README.md

```diff
@@ -133,7 +136,7 @@ or nightly from testing PyPI
 pip install -iU https://test.pypi.org/simple/ pytorch-lightning
 ```
 
-<!-- end skipping PyPI description -->
+-->
```

Comment:
Why not remove it instead?

Reply:
not sure how @edenlightning wants to formulate diff between bug-fix releases and true nightly